spiketoolkit/postprocessing/postprocessing_tools.py

```diff
@@ -1342,6 +1376,27 @@ def _select_max_channels(wf, recording, max_channels):
     return max_channel_idxs
 
 
+def _select_max_channels_from_templates(template, recording, max_channels):
```

Comment:
I don't like the method. Why a max_channels, this is not robust to channel density.
We should either:
  * have a radius
  * a threshold on peak by channel.
Or best both.

Reply:
That won't work because Phy requires all templates to have the same shape. So we have to set a number of channels

Reply:
also note that this is fine as the main goal is to find channels for each unit in which we can see differences / similarities. 16 channels are usually enough